srcpkgs/bazel/template

```diff
@@ -0,0 +1,24 @@
+# Template file for 'bazel'
+pkgname=bazel
+version=3.5.0
+revision=1
+archs="x86_64"
+wrksrc="."
+hostmakedepends="gcc openjdk11 unzip which zip"
+makedepends="python3-devel"
+short_desc="Open-source build and test tool similar to Make, Maven, and Gradle"
+maintainer="Wayne Van Son <waynevanson@gmail.com>"
+license="Apache-2.0"
+homepage="https://www.bazel.build/"
+distfiles="https://github.com/bazelbuild/bazel/releases/download/${version}/bazel-${version}-dist.zip"
+checksum=334429059cf82e222ca8a9d9dbbd26f8e1eb308613463c2b8655dd4201b127ec
+nopie_files="/usr/bin/bazel"
+
+do_install() {
+	env EXTRA_BAZEL_ARGS="--host_javabase=@local_jdk//:jdk"
+	bash ./compile.sh
```

Comment:
You need to use `export`, not `env`.

`compile.sh` seems to have a `bash` shebang already, so it isn't necessary to run it with `bash`, you can just do `./compile.sh`.

Reply:
Or, even better `EXTRA_BAZEL_ARGS="--host_javabase=@local_jdk//:jdk" ./compile.sh`.